src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1780,17 +1782,15 @@ public virtual void Dispose()
 
         public static HttpClientHandler CreateHttpMessageHandler(X509Certificate2 certificate, bool setSslProtocols, bool useCompression, bool hasExplicitlySetCompressionUsage = false)
         {
-            var httpMessageHandler = new HttpClientHandler();
+            var httpMessageHandler = new HttpClientHandler {MaxConnectionsPerServer = MinConnectionLimit };
             if (httpMessageHandler.SupportsAutomaticDecompression)
             {
                 httpMessageHandler.AutomaticDecompression =
                     useCompression ?
                         DecompressionMethods.GZip | DecompressionMethods.Deflate
                         : DecompressionMethods.None;
             }
-            else if (httpMessageHandler.SupportsAutomaticDecompression == false &&
```

Comment:
why `httpMessageHandler.SupportsAutomaticDecompression == false` was removed?

Reply:
Because the `else` handle it already, no? Or I miss here something...

Reply:
Yes, you are correct. Didn't notice. My fault.